data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1234 @@
+				accept
+
+	on accept
+		clear "gegno: landing dialog"
+		clear "gegno: asked dialog"
+		clear "gegno: escort dialog"
+		clear "gegno: deny dialog"
+		clear "gegno: there dialog"
+
+	npc
+		to spawn
+			has "gegno anticipation: quarg escort"
+		government "Quarg (Gegno)"
+		system "Heutesl"
+		personality entering staying escort
+		fleet
+			names "quarg"
+			variant
+				"Quarg Wardragon"
+
+	npc
+		on provoke
+			fail
+			"reputation: Gegno" = -100
+			"reputation: Gegno Vi" = -100
+			"reputation: Gegno Vi (Neutral)" = -100
+			"reputation: Gegno Scin" = -100
+			"reputation: Gegno Scin (Neutral)" = -100
+			dialog `After deciding to fire upon the Gegno fleet surveying your visit to their homeworld, you are met with aggressive opposition along with what sound like countless angry hails. Whatever chance you had left of associating with the Gegno is gone now.`
+		government "Gegno Vi (Neutral)"
+		personality heroic vindictive staying
+		system "Heutesl"
+		fleet "Vi Warfleet" 3
+	npc
+		on provoke
+			fail
+			"reputation: Gegno" = -100
+			"reputation: Gegno Vi" = -100
+			"reputation: Gegno Vi (Neutral)" = -100
+			"reputation: Gegno Scin" = -100
+			"reputation: Gegno Scin (Neutral)" = -100
+			dialog `After deciding to fire upon the Gegno fleet surveying your visit to their homeworld, you are met with aggressive opposition along with what sound like countless angry hails. Whatever chance you had left of associating with the Gegno is gone now.`
+		government "Gegno Scin (Neutral)"
+		personality heroic vindictive staying
+		system "Heutesl"
+		fleet "Scin Warfleet" 3
+
+
+mission "Gegno Apprehension"
+	invisible
+	landing
+	to complete
+		never
+	to offer
+		or
+			"gegno: encounters" >= 6
+			has "dead augen"
+			and
+				"gegno: encounters" >= 3
+				has "Full Vi Backlash: offered"
+			and
+				"gegno: encounters" >= 3
+				or
+					"reputation: Gegno Vi" <= -100
+					"reputation: Gegno Scin" <= -50
+			and
+				"gegno: encounters" >= 3
+				or
+					has "interrupted arena"
+					has "zydee interference"
+	on offer
+		event "gegno: apprehension"
+		fail
+
+
+mission "Gegno Intervention"
+	landing
+	clearance
+	to offer
+		has "event: gegno: apprehension"
+		not "Gegno Genocide Defense: offered"
+	source "Giaru Gegno"
+	destination "Dueyu Eitch"
+	deadline 8
+	on offer
+		fail "Scin Surveillance"
+		fail "Veering Vi A"
+		fail "Veering Vi B"
+		"reputation: Quarg (Gegno)" = 0
+		conversation
+			`While you are in the midst of docking with the ringworld, you notice a large tan-colored Gegno ship already present in your designated hangar, surrounded by several ships from both the factions of the Vi and Scin. Near them, a small group of Quarg seem to be "conversing" with a smaller group of Gegno who seem rather upset. As soon as you exit your ship, both groups stop and look your way. You recognize Adrauni swiftly approaching you alongside a Gegno that is wearing a very intricate set of armor, neither Scin nor Vi in appearance.`
```

Comment:
I believe the idea was that they were "talking" to the Gegno, when in reality, they were attempting to calm this smaller group that is more aggressively upset with you. Maybe debating would be a better word. Some form of contentious talks, rather than just a simple discussion.

Reply:
Changed it to debating in newest commit.